test/runtime/test-adapter-clone.js

```diff
@@ -0,0 +1,10 @@
+var TestAdapter = require('./test-adapter');
```

Comment:
The test adapter clone is related only to that single adapter test, maybe we should keep it inside of `test/runtime/adapter.spec.js`...